web/src/osm2streets-svelte/osm_input/BuiltInSelector.svelte

```diff
@@ -0,0 +1,75 @@
+<script lang="ts">
```

Comment:
The two `osm_input` components don't have any concept of `StreetNetwork` or osm2streets. I have some experiments where I want to process OSM data completely differently, but still conveniently extract from overpass or use something built-in